### ISSUE TITLE
When using Plex Web to cast to PKC: there was a problem selecting this stream

### ISSUE THREAD

**Issue by coreflake1** [croneter/PlexKodiConnect#357]
Hi, 
When I'm trying to stream something from the plex web interface to kodi i get the following error : 
"there was a problem selecting this stream"

I'm using Kodi 17.4 , plexkodiconnect 1.8.18 , plex 3.20.7

It keeps reappearing every +/- 5sec. 
Everything is working , the stream is playing in kodi.
But in the web interface it keeps showing this error every 5 sec.

if i look in the kodi log i get following error : 

`16:42:10.537 T:1191097248   ERROR: 192.168.0.121 - - [22/Oct/2017 16:42:10] "GET /player/playback/stop?type=video&commandID=1 HTTP/1.1" 200 -
16:42:10.838 T:1271919520   ERROR: Previous line repeats 1 times.
16:42:10.838 T:1271919520   ERROR: PLEX.plexbmchelper.httppersist: Exception encountered:
16:42:10.841 T:1271919520   ERROR: 192.168.0.121 - - [22/Oct/2017 16:42:10] "GET /player/timeline/poll?commandID=0&wait=1 HTTP/1.1" 200 -
16:42:10.843 T:1191097248   ERROR: PLEX.plexbmchelper.httppersist: Exception encountered:
16:42:10.909 T:1945440256   ERROR: Previous line repeats 1 times.
16:42:10.909 T:1945440256  NOTICE: CVideoPlayer::CloseFile()
16:42:10.910 T:1945440256  NOTICE: VideoPlayer: waiting for threads to exit
16:42:10.910 T:1945440256  NOTICE: VideoPlayer: finished waiting
16:42:10.910 T:1945440256  NOTICE: CVideoPlayer::CloseFile()
16:42:10.910 T:1945440256  NOTICE: VideoPlayer: waiting for threads to exit
16:42:10.910 T:1945440256  NOTICE: VideoPlayer: finished waiting
16:42:11.193 T:1355183008 WARNING: PLEX.librarysync: Session key 27 still unknown! Skip item
16:42:11.256 T:1355183008 WARNING: PLEX.librarysync: Session key 28 still unknown! Skip item
16:42:11.956 T:1455588256   ERROR: 192.168.0.121 - - [22/Oct/2017 16:42:11] "GET /player/timeline/poll?commandID=1&wait=1 HTTP/1.1" 200 -
16:42:13.038 T:1455588256   ERROR: 192.168.0.121 - - [22/Oct/2017 16:42:13] "GET /player/timeline/poll?commandID=1&wait=1 HTTP/1.1" 200 -
16:42:14.055 T:1455588256   ERROR: 192.168.0.121 - - [22/Oct/2017 16:42:14] "GET /player/timeline/poll?commandID=1&wait=1 HTTP/1.1" 200 -
16:42:15.102 T:1455588256   ERROR: 192.168.0.121 - - [22/Oct/2017 16:42:15] "GET /player/timeline/poll?commandID=1&wait=1 HTTP/1.1" 200 -
16:42:16.121 T:1455588256   ERROR: 192.168.0.121 - - [22/Oct/2017 16:42:16] "GET /player/timeline/poll?commandID=1&wait=1 HTTP/1.1" 200 -
16:42:17.157 T:1455588256   ERROR: 192.168.0.121 - - [22/Oct/2017 16:42:17] "GET /player/timeline/poll?commandID=1&wait=1 HTTP/1.1" 200 -
16:42:18.170 T:1141265312   ERROR: 192.168.0.121 - - [22/Oct/2017 16:42:18] "GET /player/timeline/poll?commandID=1&wait=1 HTTP/1.1" 200 -
16:42:19.186 T:1141265312   ERROR: 192.168.0.121 - - [22/Oct/2017 16:42:19] "GET /player/timeline/poll?commandID=1&wait=1 HTTP/1.1" 200 -
16:42:20.198 T:1141265312   ERROR: 192.168.0.121 - - [22/Oct/2017 16:42:20] "GET /player/timeline/poll?commandID=1&wait=1 HTTP/1.1" 200 -
16:42:21.219 T:1141265312   ERROR: 192.168.0.121 - - [22/Oct/2017 16:42:21] "GET /player/timeline/poll?commandID=1&wait=1 HTTP/1.1" 200 -
16:42:22.244 T:1141265312   ERROR: 192.168.0.121 - - [22/Oct/2017 16:42:22] "GET /player/timeline/poll?commandID=1&wait=1 HTTP/1.1" 200 -
16:42:23.263 T:1141265312   ERROR: 192.168.0.121 - - [22/Oct/2017 16:42:23] "GET /player/timeline/poll?commandID=1&wait=1 HTTP/1.1" 200 -
16:42:24.287 T:1141265312   ERROR: 192.168.0.121 - - [22/Oct/2017 16:42:24] "GET /player/timeline/poll?commandID=1&wait=1 HTTP/1.1" 200 -
16:42:25.301 T:1141265312   ERROR: 192.168.0.121 - - [22/Oct/2017 16:42:25] "GET /player/timeline/poll?commandID=1`

This is the plex log : 

`2017-10-22 16:42:15,151 (7fe175ffb700) :  DEBUG (networking:166) - Requesting 'http://192.168.0.131:3005/player/timeline/poll?commandID=1&wait=1'
2017-10-22 16:42:16,132 (7fe175ffb700) :  DEBUG (runtime:88) - Sending packed state data (99 bytes)
2017-10-22 16:42:16,133 (7fe175ffb700) :  DEBUG (runtime:924) - Response: [200] str, 339 bytes
2017-10-22 16:42:16,168 (7fe175ffb700) :  DEBUG (runtime:717) - Handling request GET /player/timeline/poll?wait=1&commandID=1
2017-10-22 16:42:16,171 (7fe175ffb700) :  DEBUG (runtime:49) - Received packed state data (80 bytes)
2017-10-22 16:42:16,174 (7fe175ffb700) :  DEBUG (runtime:814) - Found route matching /player/timeline/poll
2017-10-22 16:42:16,175 (7fe175ffb700) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/clients'
2017-10-22 16:42:16,184 (7fe175ffb700) :  DEBUG (playerservice:33) - Final remote command URL: http://192.168.0.131:3005/player/timeline/poll?commandID=1&wait=1
2017-10-22 16:42:16,186 (7fe175ffb700) :  DEBUG (networking:166) - Requesting 'http://192.168.0.131:3005/player/timeline/poll?commandID=1&wait=1'`

Does somebody knows whats happening ?

**Comment by croneter** [comment]
Please share your entire log with Kodi debug logging enabled as explained
in the PKC wiki here on github.com

Am 22.10.2017 5:25 nachm. schrieb "coreflake1" <notifications@github.com>:

**Comment by coreflake1** [comment]
here is a log file with debug logging activated.
I was watching an episode frome aprox. 21:45u till 22:02u.
The message was appearing the whole time.

thanks in advance.

[kodi.log](https://github.com/croneter/PlexKodiConnect/files/1408629/kodi.log)

**Comment by croneter** [comment]
Thanks for your logs. 

You did not follow [the instructions](https://github.com/croneter/PlexKodiConnect/wiki/How-to-Report-A-Bug), unfortunately, and thus shared your Plex token with the world. Please **immediately** follow https://forums.plex.tv/discussion/128640/reset-token

**Comment by croneter** [comment]
Something seems off, you get many of these errors: 
```
ERROR: Open - failed to open source <http://192.168.0.121:32400/photo/:/transcode?width=4000&height=4000&minSize=1&upscale=0&url=/library/metadata/21402/art/1507965150&X-Plex-Token=TOKEN>
ERROR: CCurlFile::FillBuffer - Failed: HTTP returned error 500
```
* Have you tried resetting PKC completely? (PKC Settings, Advanced, Reset the Kodi database, YES, YES, YES)
* What is your Plex Media Server log saying for these error messages?

**Comment by coreflake1** [comment]
Do i have to change the password to ? or do i simply disconnect PKC en reconnect with new pin ? 

Ontopic : 
I have tried resetting PKC completely. 
This weekend i did a fresh install of libreelec + PKC in the hope the problem would be resolved.
But without succes. 
Which plex log do you need?

**Comment by croneter** [comment]
No you don't need to reset your Plex password. Just follow the steps in the link I posted above. 

I'd need the "normal" Plex Media Server log :-)

**Comment by coreflake1** [comment]
So this isn't the Plex media server log attached above ?
When i get home, i'll post the complete zip.

**Comment by coreflake1** [comment]
[Plex Media Server Logs_2017-10-24_09-01-34.zip](https://github.com/croneter/PlexKodiConnect/files/1412424/Plex.Media.Server.Logs_2017-10-24_09-01-34.zip)

**Comment by coreflake1** [comment]
update : problem still exists on latest plex version
found some more info : 
[https://forums.plex.tv/discussion/284555/there-was-a-problem-selecting-this-stream-repeatedly-pops-up-on-on-chrome-firefox-and-safari](url)

**Comment by coreflake1** [comment]
Update : openPHT has found the bug , and has a fix 

https://github.com/RasPlex/OpenPHT/issues/268

could you implement this as well ?

**Comment by markv9401** [comment]
I still fight the same issue. Any solutions?

**Comment by croneter** [comment]
Be patient. I'm currently completely rewiring Plex Companion - a huge load
of work :-/


2018-01-03 13:44 GMT+01:00 markv9401 <notifications@github.com>:

**Comment by croneter** [comment]
Fixed with beta version 2.0.0. Guess what - you should now also be able to switch streams 😉
